contracts/validator-manager/PoSValidatorManager.sol

```diff
@@ -100,8 +100,12 @@ abstract contract PoSValidatorManager is
     error InvalidUptimeBlockchainID(bytes32 uptimeBlockchainID);
 
     // solhint-disable ordering
+    /**
+     * @dev This storage is visible to child contracts for convenience.
```

Comment:
nit: Should there be some mention of contract size in the comment? I think it is a more compelling reason than convenience.